service/src/lib.rs

```diff
@@ -271,6 +271,10 @@ pub trait Service: Send + Sync {
     ///
     /// The deployer expects this instance of [Service][Service] to bind to the passed [SocketAddr][SocketAddr].
     fn bind(&mut self, addr: SocketAddr) -> Result<(), error::Error>;
+    fn shutdown_handle(&mut self) -> Box<Option<tokio::sync::oneshot::Sender<bool>>> {
+        // TODO: get rid of the default
```

Comment:
TODO reminder :smile:

This gets hit, for example, when running the `shuttle-service` tests.

Reply:
perhaps I should leave that out :smile: 

resolving since we no longer use `fn shutdown_handle`